_readme/generate_examples.jl

```diff
@@ -8,7 +8,7 @@ function save_svg(tbl, svgfile)
         cd(dir) do
             open("input.typ", "w") do io
                 println(io, """
-                #set page(margin: 3pt, width: auto, height: auto)
+                #set page(margin: 3pt, width: auto, height: auto, fill: white)
```

Comment:
Are you able to generate two svgs each, one inverted? Then I assume https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/ might allow for just picking the most appropriate one instead of only having the white background one.

Reply:
yeah that's also possible

Reply:
ah but the stroke lines are currently not invertible in color, so I'll have to do that at a later point after this quicker fix 
<img width="353" alt="grafik" src="https://github.com/PumasAI/SummaryTables.jl/assets/22495855/7578a496-426b-47af-bbd8-142f3f7c15b9">